packages/ui/src/app/pages/Settings/Settings.tsx

```diff
@@ -69,7 +68,85 @@ export const Settings = () => {
     }
   }, [network, endpoints])
 
-  const saveSettings = () => {
+  const checkFaucetEndpoint = async () => {
+    // check faucet endpoint
+    try {
+      const faucetStatusEndpoint = customFaucetEndpoint.replace(new RegExp('register$'), 'status')
+      const response = await fetch(faucetStatusEndpoint)
+      await response.json()
```

Comment:
I think you could just check the status here:
```suggestion
      const succeeded = response.status < 400
```